es6-classes/example-class.js

```diff
@@ -0,0 +1,16 @@
+function ExampleClass() {
+
+}
```

Comment:
This should be an empty `class`.

Reply:
<img width="740" alt="Screen Shot 2022-08-30 at 11 25 01 AM" src="https://user-images.githubusercontent.com/97363006/187514633-fda13552-4089-4757-b70c-f98ee77353fc.png">